baseplate/server/monkey.py

```diff
@@ -10,4 +10,9 @@ def patch_stdlib_queues() -> None:
     import queue
     import gevent.queue
 
-    patch_module(queue, gevent.queue, items=["Queue", "LifoQueue", "PriorityQueue"])
+    monkey.patch_module(queue, gevent.queue, items=["Queue", "LifoQueue", "PriorityQueue"])
+
+
+def gevent_is_patched() -> bool:
+    """Has any gevent patching has occurred?"""
+    return bool(monkey.saved)
```

Comment:
**any** doesn't necessarily mean that it has patched what we want. What exactly does "use gevent and this works or don't and it silently fails" relate back to the usage of?

Reply:
I'm not actually sure... it's likely the case that an unpatched operation is yielded to that never yields back to gevent because it's not patched to do so. This makes it difficult to point to a single unpatched lib.

I'm not too worried about supporting beyond `patch_all` given that we never patch granularly outside of tests.